resources/Materials/TestSuite/_options.mtlx

```diff
@@ -82,5 +81,8 @@
       <!-- Suggested irradiance IBL file path -->
       <parameter name="irradianceIBLPath" type="string" value="resources/Images/san_giuseppe_bridge_diffuse.hdr" />
 
+      <!-- Transforms UVs of loaded geometry -->
+      <parameter name="transformUVs" type="matrix44" value="1.0f, 0.0f, 0.0f, -0.25f, 0.0f, 1.0f, 0.0f, 0.0f, 0.0f, 0.0f, 1.0f, 0.0f, 0.0f, 0.0f, 0.0f, 1.0f" />
```

Comment:
Added new transformUVs option to transform loaded geometry (in our case this is needed to match OSL in our tests)